src/main/java/org/jenkinsci/account/Application.java

```diff
@@ -1433,8 +1433,10 @@ public AdminUI getAdmin() {
         "vagina"
     );
 
-    public static final String SPAM_MESSAGE = "Due to the spam problem, we will need additional verification for your sign-up request. " +
-            "More details are found on <a href='https://wiki.jenkins-ci.org/display/JENKINS/User+Account+on+Jenkins'>our wiki</a> on how to get your account created.";
+    public static final String SPAM_MESSAGE = "The anti-spam system was triggered: we need your help to prove that you are human and not a robot (or a human spammer). " +
+        "Open an issue on <a href='https://github.com/jenkins-infra/helpdesk/issues'>https://github.com/jenkins-infra/helpdesk</a> and provide the email and the userid you tried to register with, as well as the version of Jenkins you currently are running. " +
```

Comment:
```suggestion
        "Open an issue on <a href='https://github.com/jenkins-infra/helpdesk/issues/new?template=1-report-issue.yml'>the Jenkins Infrastructure Help Desk</a> and provide the email and the userid you tried to register with, as well as the version of Jenkins you currently are running. " +
```

Reply:
Not sure of the Jenkins version value for these kind of issues:

```suggestion
        "Open an issue on <a href='https://github.com/jenkins-infra/helpdesk/issues/new?template=1-report-issue.yml'>the Jenkins Infrastructure Help Desk</a> and provide the email and the userid you tried to register with. " +
```

Reply:
> Not sure of the Jenkins version value for these kind of issues

It's a way to make sure that a human reads the instruction and bother enough to answer it, as I understand it.